packages/stacks-docs/product/components/textarea.html

```diff
@@ -11,13 +12,17 @@
 {% highlight html %}
 <div class="d-flex fd-column gy4">
     <label class="flex--item s-label" for="example-item">Question body</label>
-    <textarea class="flex--item s-textarea" id="example-item" placeholder="…"></textarea>
+    <div class="d-flex ps-relative">
+        <textarea class="flex--item s-textarea" id="example-item" placeholder="…"></textarea>
+    </div>
```

Comment:
We should simplify this example by removing the wrapping `div` here.

```suggestion
    <textarea class="s-textarea" id="example-item" placeholder="…"></textarea>
```

---

**Sidenote**: `.flex--item` is largely vestigial at this stage. At some point, we can deprecate the class and removing all references of it from examples but for now I just suggest removing it when updating a given component's docs.

Reply:
Hey Dan, it doesn't work without some kind of wrapper element. Can you take a look maybe? All the other examples have a wrapper div so I thought making it the "default" might be okay.
<img width="873" height="83" alt="{B9A41ADF-D580-4F95-8159-9F9EA2FC493A}" src="https://github.com/user-attachments/assets/bef26303-9021-4029-b82d-86d32c6477e0" />

Seems the `d-flex` and `ps-relative` aren't needed, though.

Reply:
Without the wrapper div the textarea doesn't respect the `width: 100%` css style. Which allows the user to resize the horizontal size of the input:
![resize](https://github.com/user-attachments/assets/574b94a5-5001-4ceb-b59f-1d1f6c638d76)

I thought it would be okay to add this wrapper here since all the other examples use it anyway. Can you take a look and let me know what you think @dancormier ?

Additionally, I removed all usages of `flex--item` from the Input and Textarea docs: https://github.com/StackExchange/Stacks/pull/2077/commits/f08863c1e21894bdd814880b01bcfb9bbb0b1ba4

Reply:
Ah, I hadn't considered that! That makes sense to wrap it.